codecov.yml

```diff
@@ -0,0 +1,2 @@
+fixes:
```

Comment:
There is a `codecov.yml` file also in : [.github/codecov.yml](https://github.com/lambdaclass/cairo-rs/blob/main/.github/codecov.yml). Maybe it can be deleted its content moved to this top level `codecov.yml` alongside your addition ?

Reply:
Yeah, I'm not sure which is preferred or whether both are being read. All I know for sure is codecov does report my `fixes` field, so it is reading that one for sure. It's not needed anyway I think, it's more a leftover from attempts to fix it following the wrong clue.

Reply:
Hey, thanks for pointing that out! It actually works without the new file :)

Reply:
Seems like it yes ! 

I wonder how it will play out whenever a file covered by tests is impacted in the PR. Indeed, in the file uploaded to codecov, I still see some `/home/runner/work/cairo-rs/` (for instance [here](https://app.codecov.io/github/lambdaclass/cairo-rs/commit/1a3b2a54e813cf2e342e111d251d6165c52dc003)). At the end, maybe codecov is perfectly fine with it and all the gymnastic with the `fixes` is not needed.

Reply:
Yes, it was likely not needed. I could try removing it.

Reply:
Oh, I see I did 🤦 
It was a long week.